### DISCUSSION TITLE
Software multitenancy

### DISCUSSION THREAD

**Discussion by carif** [mealie-recipes/mealie#1179] [Feature Request]
As I haven't found anything like that in the documentation, I'll assume this is not yet a feature, so:

**Non-technical explanation:**
You can compare this feature to something like google drive or dropbox. There is only one installation of the Dropbox system, but it handles multiple users without them noticing each other. That way, all recipes for my household can are in one space and the recipes of my friends household are in his own separate space without us noticing each other.

**Technical explanation:**
I would really like it if **a single instance** of mealie **serves multiple tenants**. That way, I could host mealie for myself, but also for the household of a friend without having to set-up two instances. 

To give some **examples of other apps**: Tandoor has this feature built in and calls it "Spaces" (thought technically, it's a multi tenant system). It's used there to separate one installation of Tandoor into several parts.

**Comment by hay-kot** [comment]
This is already baked into the v1 release - although it still has some bugs and isn't well tested since it's not a widely used feature. 

In the future, please use the feature request template

- https://github.com/hay-kot/mealie/issues/317
